scripts/presubmit.sh

```diff
@@ -110,11 +108,6 @@ main() {
       TEST_FLAGS+=" -covermode=atomic -coverprofile=coverage.txt"
     fi
 
-    if [[ ${GOVER} -ge 16 ]]; then
-      # https://github.com/google/trillian/issues/2364
-      TEST_FLAGS_SUFFIX+=" -test.paniconexit0=false"
-    fi
-
     echo "running go test ${TEST_FLAGS} ./..."
     go test ${TEST_FLAGS} ./... ${TEST_FLAGS_SUFFIX}
```

Comment:
Consider rolling this back to pre-#2365. The `TEST_FLAGS_SUFFIX` equals `-alsologtostderr` unconditionally.